e2e/playwright/projects.spec.ts

```diff
@@ -2116,3 +2116,74 @@ test(
     })
   }
 )
+
+test(
+  'segment position changes persist after dragging and reopening project',
+  { tag: '@electron' },
```

Comment:
Let's use the new tagging convention ([Slack message](https://kittycadworkspace.slack.com/archives/C04KFV6NKL0/p1748526002445369)).

```suggestion
  { tag: '@desktop' },
```